operator_test.go

```diff
@@ -499,6 +500,91 @@ math:
 		})
 	})
 
+	Convey("env Operator", t, func() {
+		op := EnvOperator{}
+		ev := &Evaluator{
+			Tree: YAML(`meta: $TEST_VAR`),
+		}
+
+		os.Setenv("TEST_VAR", "the test variable")
+		os.Setenv("SPRUCE", "awesome")
+		os.Setenv("EMPTY_VAR", "")
+
+		Convey("can resolve an empty string", func() {
+			r, err := op.Run(ev, []*Expr{str("")})
+			So(err, ShouldBeNil)
+			So(r, ShouldNotBeNil)
+			So(r.Type, ShouldEqual, Replace)
+			So(r.Value.(string), ShouldEqual, "")
+		})
+
+		Convey("can resolve a string with no environment variables", func() {
+			r, err := op.Run(ev, []*Expr{str("plain old string")})
+			So(err, ShouldBeNil)
+			So(r, ShouldNotBeNil)
+			So(r.Type, ShouldEqual, Replace)
+			So(r.Value.(string), ShouldEqual, "plain old string")
+		})
+
+		Convey("can resolve a string with a single environment variable", func() {
+			r, err := op.Run(ev, []*Expr{str("$TEST_VAR")})
+			So(err, ShouldBeNil)
+			So(r, ShouldNotBeNil)
+			So(r.Type, ShouldEqual, Replace)
+			So(r.Value.(string), ShouldEqual, "the test variable")
+		})
+
+		Convey("can resolve a string with multiple environment variables", func() {
+			r, err := op.Run(ev, []*Expr{str("with ${TEST_VAR}, spruce is ${SPRUCE}")})
+			So(err, ShouldBeNil)
+			So(r, ShouldNotBeNil)
+			So(r.Type, ShouldEqual, Replace)
+			So(r.Value.(string), ShouldEqual, "with the test variable, spruce is awesome")
+		})
+
+		Convey("handles set-but-empty, and not-set environment variables", func() {
```

Comment:
Should we have a mode where this fails if the environment variable cannot be found? I'm not sure i like the behavior of resolving these variables to "" if variables are not set, resulting in things getting accidentally unset in the manifests.